sys-fs/ncdu/ncdu-1.14.1.recipe

```diff
@@ -15,8 +15,8 @@ SECONDARY_ARCHITECTURES="x86"
 PROVIDES="
 	ncdu$secondaryArchSuffix = $portVersion
 	cmd:ncdu$secondaryArchSuffix = $portVersion
- 	commandBinDir = $binDir
 	"
+
 REQUIRES="
 	haiku$secondaryArchSuffix
 	lib:libncurses$secondaryArchSuffix
```

Comment:
Please add a single space on line 23. There should be a space between REQUIRES and BUILD_REQUIRES.